app/content/profile/index.js

```diff
@@ -300,6 +305,55 @@ function Index() {
 
   return (
     <BottomSheetModalProvider>
+      <DialogComponent
+        title={"Edit Profile Pic"}
+        content={"Upload New Pic, or Remove Current Pic"}
+        visible={uploadDialogVisible}
+        onHide={hideUploadDialog}
+        buttons={[
+          {
+            children: "Remove ",
+            pressHandler: async () => {
+              setRemoveLoading(true);
+              try {
+                await updateDoc(userRef, {
+                  pfp: "",
+                });
+                await invalidateMultipleKeys(queryClient, [["userInfo"]]);
+              } catch (e) {
+                console.log(e);
+                showDialog("Error", getErrorString(e));
+              }
+              setRemoveLoading(false);
+              hideUploadDialog();
+            },
+            loading: removeLoading,
+          },
+          {
+            children: "Upload",
```

Comment:
This may be extra credit but what do you think of this saying "Upload" when there are no pfp and "Modify" when there's one already?

Reply:
Done (I think), see updated screenshots.